<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.419, y1:0.767227, x2:1, y2:0.534, stop:0 rgba(147, 165, 207, 255), stop:1 rgba(228, 239, 233, 255))
}</string>
   </property>
   <widget class="QLabel" name="whichClass">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Sans Serif Collection&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Sans Serif Collection&quot;;
</string>
    </property>
    <property name="text">
     <string>Python:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Sans Serif Collection&quot;;
</string>
    </property>
    <property name="text">
     <string>FSD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Sans Serif Collection&quot;;
</string>
    </property>
    <property name="text">
     <string>DE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>260</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Sans Serif Collection&quot;;
</string>
    </property>
    <property name="text">
     <string>PS:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:8px;
border:3px solid rgb(139, 139, 139);
background-color:rgba(0,0,0,0);
font: 22pt &quot;Sans Serif Collection&quot;;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pythonFaculty">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:17px;
background-color: rgba(131, 185, 255,0.2);
border:3px solid rgb(139, 139, 139);
font: 18pt &quot;Sans Serif Collection&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fsdFaculty">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:17px;
background-color: rgba(131, 185, 255,0.2);
border:3px solid rgb(139, 139, 139);
font: 18pt &quot;Sans Serif Collection&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="deFaculty">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:17px;
background-color: rgba(131, 185, 255,0.2);
border:3px solid rgb(139, 139, 139);
font: 18pt &quot;Sans Serif Collection&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="psFaculty">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:17px;
background-color: rgba(131, 185, 255,0.2);
border:3px solid rgb(139, 139, 139);
font: 18pt &quot;Sans Serif Collection&quot;;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
